README.md

```diff
@@ -303,19 +308,26 @@ of data, while handling the underlying pagination logic.
 To iterate over all items, you can simply create a `Pager` instance and use it in a for loop, like this:
 
 ```python
-for branch in foundry_client.datasets.Dataset.Branch.list(dataset_rid):
-    print(branch)
+for item in client.datasets.Dataset.Branch.list(
+    dataset_rid, page_size=page_size, page_token=page_token
```

Comment:
Having page size and page token here seems to contradict the next line that "This will automatically fetch and iterate through all the pages of data".